packages/venia-concept/src/components/PurchaseDetailsPage/PurchaseDetails/__tests__/purchaseDetails.spec.js

```diff
@@ -3,11 +3,19 @@ import { configure, shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import DetailsBlock from '../../DetailsBlock';
 import PurchaseDetails from '../purchaseDetails';
+import { itemMock, otherItemsMock } from '../../itemsMocks';
 
 configure({ adapter: new Adapter() });
 
 test('renders correctly', () => {
-    const wrapper = shallow(<PurchaseDetails />).dive();
+    const fetchItemAndOtherItemsMock = jest.fn();
+    const wrapper = shallow(
+        <PurchaseDetails
+            fetchItemAndOtherItems={fetchItemAndOtherItemsMock}
+            item={itemMock()}
```

Comment:
I guess that here we need to use separate simple mocks for tests instead of using this itemMock and otherItemsMock, in the future we will need to also test rendering of the items. So just define local variable for main item and other items in this tests file

Reply:
I've added separated mocks